tests/plugins/test_add_labels.py

```diff
@@ -417,32 +417,31 @@ def test_add_labels_equal_aliases(tmpdir, docker_tasker, caplog,
         }]
     )
 
-    if isinstance(expected_log, RuntimeError):
-        with pytest.raises(PluginFailedException):
-            runner.run()
-    else:
-        runner.run()
-        assert AddLabelsPlugin.key is not None
-        result_fst = df.labels.get("description") or \
-            base_labels[INSPECT_CONFIG]["Labels"].get("description")
-        result_snd = df.labels.get("io.k8s.description") or \
-            base_labels[INSPECT_CONFIG]["Labels"].get("io.k8s.description")
-        assert result_fst == expected
-        assert result_snd == expected
+    runner.run()
+    assert AddLabelsPlugin.key is not None
+    result_fst = df.labels.get("description") or \
+        base_labels[INSPECT_CONFIG]["Labels"].get("description")
+    result_snd = df.labels.get("io.k8s.description") or \
+        base_labels[INSPECT_CONFIG]["Labels"].get("io.k8s.description")
+    assert result_fst == expected[0]
+    assert result_snd == expected[1]
 
-        if expected_log:
-            assert expected_log in caplog.text()
+    if expected_log:
+        assert expected_log in caplog.text()
 
 
 @pytest.mark.parametrize('base_fst, base_snd, base_trd, df_fst, df_snd, df_trd, expected, expected_log', [  # noqa
-    (None,  None,  None,  None,  None,  None,  None,  None),
-    (None,  None,  None,  None,  None,  'A',   'A',   'adding equal label'),
-    ('A',   'B',   'B',   None,  None,  None,  None,  RuntimeError()),
-    (None,  None,  None,  'A',   'B',   'B',   None,  RuntimeError()),
-    ('A',   'A',   'A',   None,  None,  None,  'A',   None),
-    ('A',   None,  'A',   'A',   None,  'A',   'A',   'adding equal label'),
-    ('A',   None,  None,  None,  'A',   'A',   'A',   'skipping label'),
-    (None,  'A',   'A',   'A',   'A',   None,  'A',   'skipping label'),
+    (None,  None,  None,  None,  None,  None,  (None, None, None),  None),
+    (None,  None,  None,  None,  None,  'A',   ('A', 'A', 'A'),     'adding equal label'),
```

Comment:
Putting the base and df parts into tuples as well would help readability a lot here.